### DISCUSSION TITLE
Settings Not able to be changed

### DISCUSSION THREAD

**Discussion by rywager** [AxisCommunications/acap-native-sdk-examples#224] [Q&A] [answered]
I've got an acap that had no issue using paramConfig and the structure within it to control settings from the basic app UI "3-dots" interface up until recently when i upgraded firmware to 11.9.

Worse is that all i get is the toast top right saying "Oops".  Nothing even shows up in /var/log/info.log.

Can anyone point me to what might have changed or if there are limitations i wasn't aware of?

Here's an obfuscated version but it's basically what all examples use and what mine did as well up until recently when lots of root priv stuff started to change.

            "paramConfig": [
                {
                    "name": "axisSpeakerIP",
                    "default": "192.168.128.146",
                    "type": "string"
                },
                {
                    "name": "City",
                    "default": "Kansas City",
                    "type": "string"
                },
                {
                    "name": "ELvoiceId",
                    "default": "",
                    "type": "string"
                },
                {
                    "name": "ELlanguage",
                    "default": "",
                    "type": "string"
                }
            ]

**Comment by Cacsjep** [comment]
Did u try to Compile it with newer schema Version and or sdk to see If there any issue During build, on build also manifest is checked. When there is an issue you would See it.

I Had this kind of issue also but it was my fault because I called an axparam func in the callback or other blocking things.

**Reply by rywager** [reply]
i have 1.6.0 on schema version and the same SDK as always 1.13.

Build throws no errors and i've tried changing it to force that and when there are error, they do turn up. 

Is there a limit on the settings? I see the file structure and the user it runs as, has ownership over it's directory and the file.

I don't see errors in the console or backend logging.  Where else can i check?

**Reply by Cacsjep** [reply]
Did you use a param callback? When yes, do you call a axparam function in there ?

**Reply by rywager** [reply, marked as answer]
I did not, but looks pretty interesting and is now on my list to learn for the week :).

It turned out being my app name, i had an "_" between two words and the vapix API seems to turn those into "." when you do the update on param.cgi and need the root. notation.

**Reply by Cacsjep** [reply]
Cool that you are found the issue. Normally it should set an error like "Not found" or so.
